epub33/core/vocab/item-properties.html

```diff
@@ -37,6 +37,43 @@ <h4>cover-image</h4>
 		</table>
 	</section>
 	
+	<section id="sec-exempt">
+		<h4>exempt</h4>
+		
+		<table class="tabledef" id="exempt">
+			<tbody>
+				<tr>
+					<th>Name:</th>
+					<td>
+						<code>exempt</code>
+					</td>
+				</tr>
+				<tr>
+					<th>Description:</th>
+					<td>
+						<p>The <code>exempt</code> property indicates that the [=publication resource=]
+							is an [=exempt resource=] (e.g., is used as script input or is included in the
+							[=EPUB container=] for non-rendering purposes).</p>
+						<p>If [=EPUB creators=] do not set this property, acknowledging the resources are
+							included by design, reading systems and [=EPUB conformance checkers=] may flag the
+							resources as unused if they cannot find references to them.</p>
```

Comment:
I am not sure I understand the "acknowledging the resources are included by design" remark here.